src/ngraph/frontend/onnx_import/utils/reduction.hpp

```diff
@@ -28,6 +28,7 @@
 #include "ngraph/axis_set.hpp"
 #include "ngraph/node_vector.hpp"
 #include "ngraph/op/reshape.hpp"
+#include "ngraph/op/util/arithmetic_reduction.hpp"
```

Comment:
Is this required? I see no other changes in this file? Is this PR not based on master?

Reply:
It's based on master. CLion was complaining that the `ArithmeticReduction` type wasn't declared in this file. This is used in `enable_if` in `make_ng_reduction_op` https://github.com/NervanaSystems/ngraph/blob/5e63949a9844c53dc625976a6f24fb49e4c4bad1/src/ngraph/frontend/onnx_import/utils/reduction.hpp#L74